redis/src/test/scala/zio/redis/KeysSpec.scala

```diff
@@ -112,6 +119,48 @@ trait KeysSpec extends BaseSpec {
           restored <- get(key)
         } yield assert(restore)(isRight) && assert(restored)(isSome(equalTo(value)))
       },
+      suite("migrate")(
+        testM("migrate key to another redis server (copy and replace)") {
+          for {
+            key       <- uuid
+            value     <- uuid
+            _         <- set(key, value)
+            response  <- migrate("redis1",
+                                  6379,
+                                  key,
+                                  0L,
+                                  JavaDuration.ofMillis(5000),
+                                  copy = Option(Copy),
+                                  replace = Option(Replace),
+                                  keys = None)
+            out       <- ZIO.access[SecondRedisExecutor](_.get.execute(Input.StringInput.encode("GET") ++ Input.StringInput.encode(key)))
```

Comment:
This almost works but it's a bit gross. It would be nice to use RedisCommand directly but they require a RedisExecutor not my new type. Maybe some implicit conversion could help?